pkg/virt-handler/device-manager/device_controller.go

```diff
@@ -62,73 +60,30 @@ func (c *DeviceController) nodeHasDevice(devicePath string) bool {
 	return (err == nil)
 }
 
-func (c *DeviceController) waitForPath(target string, stop chan struct{}) error {
+func (c *DeviceController) startDevicePlugin(dev GenericDevice, stop chan struct{}) {
 	logger := log.DefaultLogger()
+	deviceName := dev.GetDeviceName()
+	retries := 0
+	backoff := []time.Duration{1 * time.Second, 2 * time.Second, 5 * time.Second, 10 * time.Second}
 
-	_, err := os.Stat(target)
-	if err != nil {
-		if !os.IsNotExist(err) {
-			return err
+	for {
+		err := dev.Start(stop)
+		if err != nil {
+			logger.Reason(err).Errorf("Error starting %s device plugin", deviceName)
+			retries = int(math.Min(float64(retries+1), 3))
```

Comment:
len(backoff)-1 instead of '3'? it would be more safe.

Reply:
Done